test/libcommonserver/log/testlog.h

```diff
@@ -35,6 +35,9 @@ class TestLog : public CppUnit::TestFixture {
         log4cplus::Logger _logger;
 
         void testLog(void);
+
+    private:
+        bool _parmsDbFileExist();
```

Comment:
It's a method, should not start with `_`?

Reply:
I thought that we used the convention to prefix private function with _. I removed it (on this function and others). see commit: https://github.com/Infomaniak/desktop-kDrive/pull/100/commits/8979641ed420b2638caa3492e5965ded8c9a9a14